hogwarts-monitoring/src/main/java/org/infinispan/hp/service/CharacterSearch.java

```diff
@@ -50,8 +50,8 @@ public HPCharacter getById(String id) {
       + " WHERE c.name LIKE '%"+ term + "%'"
       + " OR c.bio LIKE '%" + term + "%'";
 
-      List<HPCharacter> characters = queryFactory.create(query).list();
-      return characters.stream().map(HPCharacter::getName).collect(Collectors.toSet());
+      List<Object> characters = queryFactory.create(query).list();
```

Comment:
since I upgraded the version, if I don't do this change it does not compile. What am I missing?

Reply:
@anistor

Reply:
@gustavonalle any clues ?

Reply:
Try this  

```List<HPCharacter> characters = queryFactory.create(query).execute().list();```